### PR TITLE
#521 Lost and Abandoned, the Tutorial, and Precision Explosives can n…

### PR DESCRIPTION
**Pull request by Favid**
…o longer be enabled

### CODE REVIEW

File: LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIListener_CampaignStartMenu.uc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    if (UIShellDifficulty(Screen) != none)
+    {
+        ShellDifficulty = UIShellDifficulty(Screen);
+        ShellDifficulty.m_TutorialMechaItem.Checkbox.SetChecked(false);
+        ShellDifficulty.m_TutorialMechaItem.SetDisabled(true, strDisabledTutorialTooltip);
+
+		// Disables Advanced Option for Precision Explosives
+		// Hard-coding the index should be fine because Firaxis already relies on these indexes being hard-coded
+		UIMechaListItem(ShellDifficulty.m_SecondWaveList.GetItem(7)).Checkbox.SetChecked(false);
+		UIMechaListItem(ShellDifficulty.m_SecondWaveList.GetItem(7)).SetDisabled(true, strDisabledPrecisionExplosivesTooltip);
```
Comment (pledbrook) [inline LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIListener_CampaignStartMenu.uc]: Would you mind using tabs for indent throughout the file, please?

**Comment by pledbrook** [comment]
Oh, I also highly recommend adding a "Resolves #521" in the commit message. GitHub will automatically close the issue with that number when the commit is merged into `master`. See [this GitHub article](https://help.github.com/en/articles/closing-issues-using-keywords) for more info.

File: LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIListener_CampaignStartMenu.uc (hunk 1 above)
Comment (Favid) [reply LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIListener_CampaignStartMenu.uc]: That's what I get for copying code. Done.

**Comment by alexu4326** [comment]
so, now we can play with lost and abandoned ?

**Comment by Clibanarius** [comment]
The opposite. You shouldn't have been playing with it before and now you simply cannot.

**Pull request merged**